### DISCUSSION TITLE
Reverse vnc support

### DISCUSSION THREAD

**Discussion by inventron-tr** [any1/neatvnc#128] [Q&A]
As you know after Weston 12 we can enable VNC with neatvnc support. Is there a possibility to have reverse vnc support where the server will initiate the connection to client. Thanks.

**Comment by any1** [comment]
This is not a feature. Can you give an example of how this might be used?

**Comment by inventron-tr** [comment]
This is actually an essential feature where the server is behind a firewall or corporate network. Then the device or computer running the vnc server initiates the connection to the computer that will control this device or computer. Like the one in https://help.ubuntu.com/community/VNC/Reverse

**Reply by any1** [reply]
I recommend that you create a PR for this. Please read CONTRIBUTING.md before you do.
